### PR TITLE
add html and servlet for form feature

### PR DESCRIPTION
**Pull request by IvanSotomayor**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DateTimeServlet.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+
+/** Servlet that responds with the current date. */
+@WebServlet("/dateTime")
+public class DateTimeServlet extends HttpServlet {
+
+    DateTimeFormatter dtf = DateTimeFormatter.ofPattern("uuuu/MM/dd");
+    LocalDate localDate = LocalDate.now();
```
Comment (roberriSPS) [inline portfolio/src/main/java/com/google/sps/servlets/DateTimeServlet.java]: The `LocalDate localDate = LocalDate.now();` and `LocalTime localTime = LocalTime.now();` can be done inside the doGet. That way the date and time can be an updated time everytime the get is called.

**Review by roberriSPS — CHANGES_REQUESTED** [review]
LGTM just the unresolved comment

File: portfolio/src/main/java/com/google/sps/servlets/DateTimeServlet.java (hunk 1 above)
Comment (IvanSotomayor) [reply portfolio/src/main/java/com/google/sps/servlets/DateTimeServlet.java]: Done in my last commit 👍

**Review by roberriSPS — APPROVED** [review]
LGTM

**Pull request merged**